articles/openshift/oshift-how-portworx-snapshots.md

```diff
@@ -0,0 +1,347 @@
+---
+title: How to create and restore Portworx snapshots | UKCloud Ltd
+description: Explains how to create and restore the different types of Portworx snapshots
+services: openshift
+author: Kieran O'Neill
+reviewer:
+lastreviewed: 15/10/2019
+
+toc_rootlink: How To
+toc_sub1: Use Portworx with OpenShift
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Expose non-http services externally
+toc_fullpath: How To/Use Portworx with OpenShift/oshift-how-portworx-snapshots.md
+toc_mdlink: oshift-how-portworx-snapshots.md
+---
+
+# How to create and restore Portworx snapshots
+
+## Overview
+
+Portworx is a cloud-native storage solution that is now available as an add-on to our OpenShift offering. This article describes how to use Portworx within OpenShift to create on-demand or scheduled snapshots of persistent volumes and how to restore them.
+
+### Intended audience
+
+This article assumes you have access to a Portworx-enabled OpenShift 3.11 cluster and that you have cluster-admin rights. It also assumes familiarity with `oc`, the OpenShift command-line client. 
+
+If you're interested in a free 30 day trial of Portworx, raise a Service Request via the [My Calls](https://portal.skyscapecloud.com/support/ivanti) section of the UKCloud Portal.
+
+## Creating on-demand snapshots
+
+### Same namespace
+
+You can take on-demand snapshots by creating an OpenShift resource of type `VolumeSnapshot` using the `oc` command line tools.
+
+1. Assuming you have a PVC named `test-snapshot-pvc`, provisioned by Portworx in the `test-snapshot-1` namespace, to snapshot this you would run the following:
```

Comment:
Rather than just stating some assumptions here can we create a pre-req's section further up, even if we refer to another doc on how to create volumes backed by Portworx